packages/single-invoice/src/utils/debounce.d.ts

```
@@ -0,0 +1 @@
+export declare const debounce: (func: Function, wait: number) => (...args: any[]) => void;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)